code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -218,10 +218,16 @@
 	force_threshold = 10
 	pressure_resistance = 50
 	mob_size = MOB_SIZE_LARGE
-	see_invisible = SEE_INVISIBLE_MINIMUM
 	see_in_dark = 8
 	var/independent = FALSE
 
+/mob/living/simple_animal/hostile/blob/blobbernaut/Login()
+	. = ..()
+	if (hud_used)
```

Comment:
I'd prefer if we had some kind of replacement for ```see_invisible```, and all of this boilerplate was in one place (```/mob/Initialize()```? ```/datum/hud/initialize()```?)